src/parser.nim

```diff
@@ -61,13 +61,15 @@ proc parseGraphList*(js: JsonNode): List =
 
   result = List(
     id: list{"id_str"}.getStr,
-    name: list{"name"}.getStr.replace(' ', '-'),
+    name: list{"name"}.getStr,
     username: list{"user", "legacy", "screen_name"}.getStr,
-    userId: list{"user", "legacy", "id_str"}.getStr,
+    userId: list{"user", "rest_id"}.getStr,
```

Comment:
Are you sure about this one? I remember there being some weirdness going on with id_str vs rest_id

Reply:
According to my observation , there's no `id_str` under `legacy` now. so i had to choose `rest_id` to fill userId.